config/config.yaml

```diff
@@ -8,6 +8,9 @@ reference:
   build: GRCh38
   # Ensembl release
   release: 107
+  # flavor: ""  # optional, e.g. chr_patch_hapl_scaff, see Ensembl FTP.
```

Comment:
Can we get a(n example) link here? This always makes `config.yaml` adjustments so much easier...

Reply:
Yes, that would be good - I simply copied the comments from the wrapper, because I have no experience with different flavors myself ;)